entrypoint.sh

```diff
@@ -92,12 +91,15 @@ ansible::lint() {
 
   # Enable recursive glob patterns, such as '**/*.yml'.
   shopt -s globstar
-  ansible-lint -v --force-color $opts ${TARGETS}
+  ansible-lint -v --force-color $opts ${INPUT_TARGETS=}
```

Comment:
Did you mean this?
```suggestion
  ansible-lint -v --force-color $opts ${INPUT_TARGETS:=}
```
My google-fu failed to reveal how just `=` works.

Reply:
This is a good documentation: https://tldp.org/LDP/abs/html/parameter-substitution.html

Using `:=` here shouldn't matter because we are replacing the variable with the empty string.

We actually only need this because we have `set -u` at the start of the script, making referencing undefined variables an error.